pom.xml

```diff
@@ -87,11 +87,11 @@
         <gogo.jline.version>1.0.6</gogo.jline.version>
         <slf4j.version>1.7.21</slf4j.version>
 
-        <surefire.argLine />
+        <surefire.argLine/>
     </properties>
 
     <prerequisites>
-        <maven>2.2.1</maven>
+        <maven>3.5.0</maven>
```

Comment:
Fwiw, this line is of no use.  Maven 3 does not use it according to https://maven.apache.org/pom.html#Prerequisites, we should use the enforcer plugin for that and also specifies JDK8.